frontend/src/store/modules/v1/project.ts

```diff
@@ -40,12 +40,25 @@ export interface ProjectFilter {
 
 const getListProjectFilter = (params: ProjectFilter) => {
   const list = [];
-  const search = params.query?.trim().toLowerCase();
+  const search = params.query?.trim();
+
   if (search) {
-    list.push(
-      `(name.matches("${search}") || resource_id.matches("${search}"))`
-    );
+    // Check if the search contains label filtering syntax
```

Comment:
It would be better if we could add another field like `lables: {key: string; value: string}[]` for the `ProjectFilter`